EIPS/eip-6077.md

```diff
@@ -0,0 +1,73 @@
+---
+eip: 6077
+title: Invalidation abstraction for signature-based operations powered by EIP-712
+description: Extends EIP-712 and unifies EIP-2612 with many others
+author: Anton Bukov (@k06a), Mikhail Melnik (@zumzoom)
+discussions-to: TBD
```

Comment:
Needs to be a thread on ethereum magicians

Reply:
Thx, added: https://ethereum-magicians.org/t/eip-6077-invalidation-abstraction-for-signature-based-operations/12162